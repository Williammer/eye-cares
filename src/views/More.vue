<template>
  <h2 class="more">
    More eye care games are coming soon...
  </h2>
</template>

<style lang="scss">
.more {
  margin-top: 20px;
  color: #555;
}
</style>
