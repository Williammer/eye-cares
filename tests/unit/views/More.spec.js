import { shallowMount } from '@vue/test-utils';
import More from '@/features/more/More.vue';

describe('More.vue', () => {
  it('should have the expected DOM', () => {
    const wrapper = shallowMount(More);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
