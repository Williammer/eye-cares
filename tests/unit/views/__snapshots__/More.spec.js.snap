// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`More.vue should have the expected DOM 1`] = `
<h2 class="more">
  More eye care games are coming soon...
</h2>
`;
